lessons/pydata/pandas_core/index.ipynb

```diff
@@ -0,0 +1,3402 @@
+{
```

Comment:
<p>Vynechal bych u nadpisu to <code>.values</code>, je to napsané o řádek níž. Takhle je to trochu kryptické.</p><p>Stejně tak u dalších nadpisů.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Ok.</p>